Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -1691,12 +1691,12 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
     }
     
     /// - returns: The DataSet object displayed at the touched position of the chart
-    @objc open func getDataSetByTouchPoint(point pt: CGPoint) -> IBarLineScatterCandleBubbleChartDataSet!
+    @objc open func getDataSetByTouchPoint(point pt: CGPoint) -> IBarLineScatterCandleBubbleChartDataSet?
```

Comment:
is this warned by Xcode 9.3? or why we change

Reply:
It's never been correct to use IUO as a return type or parameter type. This is deprecated in 4.1 and will be an error in a future release. [SE-0054](https://github.com/apple/swift-evolution/blob/master/proposals/0054-abolish-iuo.md), [SR-3023](https://bugs.swift.org/browse/SR-3023)